TestFoundation/TestNSTimeZone.swift

```diff
@@ -117,6 +117,20 @@ class TestNSTimeZone: XCTestCase {
         let known = NSTimeZone.knownTimeZoneNames
         XCTAssertNotEqual([], known, "known time zone names not expected to be empty")
     }
+    
+    func test_localizedName(){
```

Comment:
I've got a few nits myself before I'm ready to merge, if you don't mind. `test_localizedName(){` → `test_localizedName() {`